Frontend/ui-driver/src/Strings/ML.js

```diff
@@ -468,5 +468,13 @@ const mlStrings = {
   COMPLAINTS_DESCRIPTION : "എന്തെങ്കിലും പരാതികൾ ഉണ്ടെങ്കിൽ, <u>nammayatri.support@juspay.in</u> എന്ന വിലാസത്തിൽ ഞങ്ങളെ ബന്ധപ്പെടുക;",
   COMPLAINTS_DESCRIPTION_ADDITIONAL : "പരാതികൾ പരിഹരിക്കുന്നതിന്, ഞങ്ങളുടെ <u>സ്വകാര്യതാ നയം</u> പരിശോധിക്കുക",
   REGISTERED_ADDRESS : "രേഖപ്പെടുത്തിയ വിലാസം",
-  REGISTERED_ADDRESS_DESCRIPTION : "Stallion Business Centre, No. 444, 3rd & 4th Floor, 18th Main, 6th Block, Koramangala, Bengaluru, Karnataka- 560095, India"
+  REGISTERED_ADDRESS_DESCRIPTION : "Stallion Business Centre, No. 444, 3rd & 4th Floor, 18th Main, 6th Block, Koramangala, Bengaluru, Karnataka- 560095, India",
+  SELECT_YOUR_GENDER : "നിങ്ങളുടെ ലിംഗഭേദം തിരഞ്ഞെടുക്കുക",
+  FEMALE : "സ്ത്രീ",
+  MALE : "പുരുഷൻ",
+  PREFER_NOT_TO_SAY : "പറയാൻ ആഗ്രഹിക്കുന്നില്ല",
+  GENDER : "ലിംഗഭേദം",
+  SET_NOW : "Set Now",
```

Comment:
Please add translations wherever required

Reply:
Sorry, I don't know how few were left in Malayalam.
Fixed it now.